src/pages/index.md

```diff
@@ -36,6 +36,28 @@ sections:
           - label: Partner With Us
             url: /organizations
             style: secondary
+      - title: Projects
+        bgcolor: e8edf3
+        justify-content: flex-start
+        cards:
+          - image: /images/Rectangle-19.png
+            alt: Phone taking screenshot of Nike hoodie
+            label:  Project A
+            caption: Web Development
+            redirect: http://localhost:8888/projects
```

Comment:
should the redirect not be: `/projects`?

Reply:
read more on absolute vs relative urls: https://docs.microsoft.com/en-us/sql/ado/guide/data/absolute-and-relative-urls?view=sql-server-ver16